pix-editor/app/serializers/theme.js

```diff
@@ -5,6 +5,7 @@ export default class ThemeSerializer extends AirtableSerializer {
 
   attrs = {
     name: 'Nom',
```

Comment:
Refacto à faire : renommer côté Airtable
```suggestion
    name: 'Titre fr-fr',
```

Reply:
mis en issue